cmds/provisiond/main.go

```diff
@@ -139,6 +145,7 @@ func main() {
 		Statser:        statser,
 		ZbusCl:         zbusCl,
 		Janitor:        provision.NewJanitor(zbusCl, puller),
+		MemStats:       memStats,
```

Comment:
You don't have to pass it, I think the engine can get that internally and keep it.